source/connect/stable-api.txt

```diff
@@ -0,0 +1,24 @@
+.. _scala-stable-api:
+
+===========
+Stable API
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: compatible, backwards, upgrade
```

Comment:
[s] Might be worth updating the keywords now so it doesn't get missed when this page is written.

Reply:
Stable API page already completed!